### ISSUE TITLE
API is not working anymore

### ISSUE THREAD

**Issue by andreepratama27** [tomorisakura/unofficial-masakapahariini-api#19]
Hi Guys,

I found that the API is not working anymore. It will return empty response, for example in this endpoint [here](https://masak-apa.tomorisakura.vercel.app/api/recipes).

Could you guys help to check? Thank you

**Comment by tomorisakura** [comment]
yeah, it will be updated soon because their main website already updated with new layout. it a reason why you got empty response.

**Comment by andreepratama27** [comment]
Thanks for your response, @tomorisakura
